tests/helpers/skipif.py

```diff
@@ -21,52 +21,56 @@
 from pytorch_lightning.utilities import _TORCH_QUANTIZE_AVAILABLE
 
 
-def skipif_args(
-    min_gpus: int = 0,
-    min_torch: Optional[str] = None,
-    quant_available: bool = False,
-) -> dict:
-    """ Creating aggregated arguments for standard pytest skipif, sot the usecase is::
-
-        @pytest.mark.skipif(**create_skipif(min_torch="99"))
-        def test_any_func(...):
-            ...
+class SkipIf:
+    """
+    SkipIf wrapper for simple marking specific cases, fully compatible with pytest.mark::
 
-    >>> from pprint import pprint
-    >>> pprint(skipif_args(min_torch="99", min_gpus=0))
-    {'condition': True, 'reason': 'Required: [torch>=99]'}
-    >>> pprint(skipif_args(min_torch="0.0", min_gpus=0))  # doctest: +NORMALIZE_WHITESPACE
-    {'condition': False, 'reason': 'Conditions satisfied, going ahead with the test.'}
+        @SkipIf(min_torch="0.0")
+        @pytest.mark.parametrize("arg1", [1, 2.0])
+        def test_wrapper(arg1):
+            assert arg1 > 0.0
     """
-    conditions = []
-    reasons = []
 
-    if min_gpus:
-        conditions.append(torch.cuda.device_count() < min_gpus)
-        reasons.append(f"GPUs>={min_gpus}")
+    def __new__(
+        self,
+        *args,
+        min_gpus: int = 0,
+        min_torch: Optional[str] = None,
+        quantization: bool = False,
+        **kwargs
+    ):
+        conditions = []
```

Comment:
docstring would be nice :]
```suggestion
        """
        Args:
            args: ?
            num_gpus: min number of gpus required to run test
            min_torch: minimum pytorch version to run test
            quantization: if `torch.quantization` package is required to run test
            kwargs: ?
        """
        conditions = []
```